emu/src/timer_registers.rs

```diff
@@ -0,0 +1,132 @@
+use crate::bitwise::Bits;
+
+pub enum TimerRegisterAccessControl {
+    Read,
+    Write,
+    ReadWrite,
+}
```

Comment:
I think it would be better to define an enum like `IoRegisterAccessControl` and use it both here and in `lcd_registers` and future IO registers, instead of creating new enums every time.

Reply:
Perfect!

Reply:
Agreed